jbrowse/src/client/JBrowse/VariantTable/constants.tsx

```diff
@@ -86,7 +86,7 @@ export const columns: GridColumns = [
   { field: 'impact', headerName: 'Impact', width: 50, type: "string", flex: 1, headerAlign: 'left' },
   { field: 'overlapping_genes', headerName: 'Overlapping Genes', type: "string", flex: 1, headerAlign: 'left' },
   { field: 'cadd_ph', headerName: 'CADD Score', width: 50, type: "number", flex: 1, headerAlign: 'left' },
-  { field: 'track_id', headerName: 'Track ID', width: 50, type: "string", flex: 1, headerAlign: 'left', hide: true },
-  { field: 'start', headerName: 'Start Location', width: 50, type: "string", flex: 1, headerAlign: 'left', hide: true },
-  { field: 'end', headerName: 'End Location', width: 50, type: "string", flex: 1, headerAlign: 'left', hide: true }
+  { field: 'track_id', headerName: 'Track ID', width: 50, type: "string", flex: 1, headerAlign: 'left', hide: true, filterable: false},
```

Comment:
I agree these might not be that important to filter on, but is there a reason to disallow it for start/end?

Reply:
I was disabling filtration for the columns that are default hidden, just for consistency's sake. It might just make sense to enable them all instead, rolling back these lines, if we want some of them to be filterable

Reply:
Is there some operational or aesthetic reason a hidden column shouldnt be filterable? does filter UI show up in weird places when it's hidden?

Reply:
The only thing is that if user filters by a currently-hidden column, it's possible to get the table into a state where it has less data than expected (or none at all) without much visible feedback to the user. Thinking about it more, though, this will be handled in #156, so I'll remove these changes and we can handle this problem there.